core/src/main/resources/lib/layout/html.jelly

```diff
@@ -37,8 +37,7 @@ THE SOFTWARE.
       Title of the HTML page. Rendered into &lt;title> tag.
     </st:attribute>
     <st:attribute name="norefresh">
```

Comment:
@daniel-beck As far as I remember, this Jelly tag is deprecated and suggested for removing (https://github.com/jenkinsci/jenkins/pull/2852)